Add Git Server tool (#9265)
kyma-project/kyma: tools/gitserver/README.md

# Git Server

> This is a tool that exposes an HTTP server with git repository, that can be used in e2e tests, to avoid connections with external git providers.

Reviewer: ```suggestion
Git Server is a tool that exposes an HTTP server with a Git repository. It can be used in e2e tests to avoid connections with external Git providers.
```

> Additional git repositories can be added simply by creating new directory in `/repos/`.

Reviewer: ```suggestion
You can add additional Git repositories simply by creating a new directory in `/repos/`.
```
